### ISSUE TITLE
add support for LDA/GGA+U

### ISSUE THREAD

**Issue by bernstei** [SPARC-X/SPARC#45]
Many systems of interest involve transition metals that are better described by LDA+U or GGA+U, so it would be nice to support those sorts of functionals (also hybrid functionals, but I gather those are already being planned, right?)

**Comment by phanish-suryanarayana** [comment]
Yes, LDA/GGA+U is another on our list of features to be incorporated. Hybrid functionals have already been implemented and tested. It should be released in the coming weeks.
